### PR TITLE
Infra: Add manuzhang to collaborators

### PR DESCRIPTION
**Pull request by manuzhang**
I'm volunteering to be a collaborator while Brian(@bitsondatadev) embarks on his new journey. After offline discussion, he graciously agreed to be removed from the collaborator list.

### CODE REVIEW

**Comment by manuzhang** [comment]
Most CIs shouldn't be triggered by this change. I submitted #11860 to fix.

File: .asf.yaml [hunk 1]
```diff
@@ -54,7 +54,7 @@ github:
     - SreeramGarlapati
     - samredai
```
Comment (bitsondatadev) [inline .asf.yaml:55]: Actually, if all is the same to @samredai, would you mind swapping his name instead? I still plan on doing some docs work but didn't want to hold down the list, but I'm pretty sure Sam is on to other things.

If this is too much of a bother then happy to merge this now and I'll add a new PR later to swap with Sam's name when I need it.

**Review by bitsondatadev — APPROVED** [review]
Added a comment, I'm fine if this merges and I need to add another PR later.

File: .asf.yaml (hunk 1 above)
Comment (manuzhang) [reply .asf.yaml:55]: Oh, I might have misunderstood your words. Then let me cancel this PR.

Comment (samredai) [reply .asf.yaml:55]: Not a problem! Please feel free to swap my name out instead. I haven't been as active as I'd like lately so this makes sense to me.

**Pull request closed without merging**